packages/form-config/locales/eusm/ar.json

```diff
@@ -1,25 +1,34 @@
 {
-  "Make Release": "",
+  "Action": "",
+  "App ID": "",
+  "App Version": "",
+  "Click to upload": "",
+  "Created at": "",
+  "Download": "",
+  "Draft Files": "",
+  "Edit": "",
+  "Error occurred": "",
   "File Name": "",
   "File Version": "",
-  "Identifier": "",
-  "Module": "",
-  "Edit": "",
-  "Download": "",
-  "Upload edit": "",
-  "Upload New File": "",
-  "View Files": "",
-  "App Id": "",
-  "App Version": "",
-  "Related to": "",
-  "Upload file": "",
-  "Form is required": "",
-  "Form name is required": "",
   "Find Draft Files": "",
   "Find Files": "",
   "Find Release": "",
+  "Form": "",
+  "Form is required": "",
+  "Form Name": "",
+  "Form Name is required": "",
+  "Identifier": "",
+  "JSON Validators": "",
+  "Make Release": "",
+  "Module": "",
+  "Related to": "",
+  "Releases": "",
   "Search": "",
-  "Created at": "",
   "Updated at": "",
-  "Error occurred": ""
+  "Upload edit": "",
```

Comment:
Is this right?

Reply:
@ciremusyoka It's the same string currently in use
![Screenshot from 2021-03-02 11-59-20](https://user-images.githubusercontent.com/19818819/109624037-cc4a0280-7b4e-11eb-9960-7f887bd0ff9f.png)